src/flb_utils.c

```diff
@@ -535,7 +535,6 @@ int flb_utils_write_str(char *buf, int *off, size_t size,
         return FLB_FALSE;
     }
 
-    written = *off;
```

Comment:
I think this is correct. written aims to count the number of bytes written on this call without count the previously written, otherwise it could mess up the offset [here](https://github.com/fluent/fluent-bit/blob/master/src/flb_utils.c#L645).

I cannot see any anomaly or bad-side effect on this change.